drivers/input/Kconfig.gt911

```diff
@@ -26,4 +26,10 @@ config INPUT_GT911_INTERRUPT
 	help
 	  Enable interrupt support (requires GPIO).
 
+config INPUT_GT911_TOUCH_NUMBER
```

Comment:
naming sounds a bit weird to me, maybe `INPUT_GT911_MAX_TOUCH_POINTS`?

Reply:
Changed! I initially wanted to keep the same register name but it's not very telling indeed.